woocommerce/woocommerce-ios
# [Order Fulfillment][Part 3] Add customer section to Review Order screen

Part 3 of #4136 
For more context there's [part 1](https://github.com/woocommerce/woocommerce-ios/pull/4534), [part 2](https://github.com/woocommerce/woocommerce-ios/pull/4535), [part 4](https://github.com/woocommerce/woocommerce-ios/pull/4549), [part 5](https://github.com/woocommerce/woocommerce-ios/pull/4567).

# Description
This PR follows up #4535 to add new section Customer Information to the Review Order screen.

# Solution
- ReviewOrderViewModel was updated to config rows for Customer section. This section should contains the following rows:
  - Customer note if order has additional note
  - Shipping address if order has at least one non-virtual product
  - Shipping method if order has shipping info
  - View billing info
- ReviewOrderViewController was updated to config styling and content for cells based on row information.

# Screenshots
| Customer section with note only | Customer section with Shipping Details |
|----|----|
|<img src="https://user-images.githubusercontent.com/5533851/124456843-621fc380-ddb5-11eb-8844-9a94511007ab.png" width=400 />|<img src="https://user-images.githubusercontent.com/5533851/124456848-63e98700-ddb5-11eb-91c5-f1835980281d.png" width=400 />|

# Testing
- Navigate to Orders tab
- Select an order with In Progress status
- Select Mark Order Complete
- You should then be navigated to Review Order screen with two sections Product and Customer Information.

Update release notes:
